mappings/net/minecraft/item/ItemPropertyGetter.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS awn net/minecraft/item/ItemPropertyGetter
 	METHOD call (Lawo;Lbbp;Lahe;)F
-		ARG 1 world
-		ARG 2 user
+		ARG 1 stack
+		ARG 2 world
+		ARG 3 user
```

Comment:
Not particularly relevant to this PR, but what is `user`?

Reply:
It seems for clocks and compasses it is used to get additional information like the world and the orientation of the user.